### DISCUSSION TITLE
Register Interest in Upcoming Requests/Shows

### DISCUSSION THREAD

**Discussion by Hampo** [sct/overseerr#3792] [Feature Requests]
At the moment, one person requests an item, and they're the only one who can get notifications when it's released. The ability to "register interest" or "subscribe" to a Pending/Approved/Processing request would add you to the notification list when they're sent. IE, include their ping in the Discord message or also send them an email.
A similar feature would be nice for TV shows, to be notified when new episodes are available.

**Comment by Storeyv34** [comment]
https://github.com/sct/overseerr/issues/789
